applications/DEM_application/custom_strategies/strategies/explicit_solver_strategy.cpp

```diff
@@ -489,10 +489,10 @@ namespace Kratos {
         GetForce(); // Basically only calls CalculateRightHandSide()
         //FastGetForce();
         GetClustersForce();
+        GetRigidBodyElementsForce();
```

Comment:
Is this implying we are now calculating the rigidbodyforces (always) even if we do not want to activate the COMPUTE_FEM_RESULTS_OPTION ?

Reply:
That change is correct and in fact was a bug. Otherwise, the rigid bodies did not move at all as a default.

Reply:
Yes, as @salvalatorre said if we want the rigid bodies to move when contacting other particles we should do it. If we impose Linear/Periodic movement we do not need to calculate those forces, how can we improve this?

Reply:
I was commenting on that because in a particular case (spreaddem) we want to avoid as much as possible unnecessary computations and in that case in particular there wont be any FEM moving cause of particles impacting on it (only the other way around). not sure how we could improve this atm.